bin/fetchdata.js

```diff
@@ -20,6 +20,10 @@ function getAttributeNames(attributeList) {
         return attributeList.data.map((attr) => attr.attributes.name.toLowerCase());
     }
 
+    if (attributeList.data === null) {
```

Comment:
This removes the error but doesn't actually solve the problem, which is that we need to generate a correct json file even if the input data is formatted incorrectly. Right now this creates the token files with no card types, which will cause problems when we implement those cards. It looks like the Palpatine / Sidious leader is also missing some basic data that will cause issues.

Unfortunately we may need to add a helper method that checks for special cases and manually fills in values where we know they're missing.

Reply:
What did you see on palp ? To me I do not see anything missing. Probably some null property will break something but it's a weird card 🤷‍♂️

Reply:
Can we remove this check now, I think it's important that if we a property is null it will throw an error. Otherwise we wouldn't have caught this issue in the first place.